cmake/MiNiFiOptions.cmake

```diff
@@ -42,6 +42,8 @@ add_minifi_option(DOCKER_SKIP_TESTS "Skip building tests in docker image targets
 add_minifi_option(DOCKER_PUSH "Push created images to the specified tags" OFF)
 
 add_minifi_option(PORTABLE "Instructs the compiler to remove architecture specific optimizations" ON)
+add_minifi_option(USE_CONAN_PACKAGER "Instructs cmake to use conan packager to install external libs & build MiNiFi C++" OFF)
```

Comment:
Since there is only two exclusive options as of now, it doesnt make sense to create two separate options here.

I suggest (for now) to only have a single MINIFI_USE_CONAN_PACKAGER option, and replace the if elseif with a simple if else in the cmake files

And please prefix the new option with MINIFI_ (it makes easier to differentiate our options with thirdparty ones)

Reply:
I incorporated your feedback into commit [cf2face](https://github.com/apache/nifi-minifi-cpp/pull/1793/commits/cf2facee3a1ecc2909d439a4fe9cd1ca60830617)